src/SDKs/CognitiveServices/dataPlane/Language/LUIS-Runtime/Microsoft.Azure.CognitiveServices.LUIS.csproj

```diff
@@ -14,4 +14,4 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
-</Project>
+</Project>
```

Comment:
@pcostantini this file is missing changes from upstream. Seems like not merged correctly with changes that are in upstream.

Reply:
Hi @shahabhijeet ,
Rebased recent changes, should contain all upstream changes now.
Regards,
-PabloC.